### PR TITLE
Add constants for watchOS and tvOS

### LINKED ISSUE #323: Add constant specs for watchOS and tvOS
@mrackwitz any guidance you have on how to do this would be much appreciated!

### PR DESCRIPTION
**Pull request by segiddins**
Closes #323.

### CODE REVIEW

**Comment by segiddins** [comment]
\c @mrackwitz @neonichu will merge this unless there are any objections

File: spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__ [hunk 1]
```diff
@@ -0,0 +1 @@
+Did you know that git does not support storing empty directories?
```
Comment (AliSoftware) [inline spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__]: Was this file auto-generated? Aren't `.gitkeep` files the commonly accepted practice for cases like this instead?

File: spec/spec_helper/project_helper.rb [hunk 2]
```diff
@@ -4,10 +4,13 @@ module SpecHelper
   module ProjectHelper
     # Keys which are excluded from comparison
     EXCLUDED_KEYS = %w(
+      IBSC_MODULE
```
Comment (AliSoftware) [inline spec/spec_helper/project_helper.rb]: Just out of curiosity, does anyone know what this build setting is usually for?

File: spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__ (hunk 1 above)
Comment (segiddins) [inline spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__]: This is what Xcode generates

File: spec/spec_helper/project_helper.rb (hunk 2 above)
Comment (segiddins) [inline spec/spec_helper/project_helper.rb]: Yes -- it's what determines the extension the watchOS app is associated with

File: spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__ (hunk 1 above)
Comment (mrackwitz) [inline spec/fixtures/CommonBuildSettings/Project/Objc_watchOS_Native Extension/Assets.xcassets/README__ignoredByTemplate__]: This seems to be an issue with either the template or the template generation. However for now it should stay here.

**Comment by mrackwitz** [comment]
Great work! Thanks. :+1:

**Pull request merged**